pkg/flow/database/entwrapper/database.go

```diff
@@ -186,6 +186,18 @@ func New(ctx context.Context, sugar *zap.SugaredLogger, addr string) (*Database,
 	 		    CONSTRAINT "fk_namespaces_mirror_processes"
 				FOREIGN KEY ("namespace_id") REFERENCES "namespaces"("oid") ON DELETE CASCADE ON UPDATE CASCADE
 	 		);
+	 CREATE TABLE IF NOT EXISTS "secrets" 
```

Comment:
Unlike other tables in our refactor, I don't think we ever need a UUID for a secret. How do you feel about deleting this column and making a compound primary key out of `namespace_id` and `name`?

Reply:
I thought about that, but we need to stick to the schema and don't change it if its not hugely necessary. WDYT?

Reply:
Good point.